sample_apps/run_example.sh

```diff
@@ -1290,11 +1587,19 @@ function run_simple_app_under_gramine_as_client_make_trusted_request() {
     echo "${Me}: $(TZ="America/Los_Angeles" date) Completed ${FUNCNAME[0]} ${DryRun_msg}."
 }
 
+# ###########################################################################
+function run_simple_app_under_sev_as_client_make_trusted_request() {
```

Comment:
Relocate to L1534.

Reply:
Done. relocated call to just before callee, so it's clear that both `simple_app` and `simple_app_under_sev` run the same shared function.